src/model/presentation/xml/StateDelegate.java

```diff
@@ -0,0 +1,10 @@
+package model.presentation.xml;
+
+import model.Pencil;
+
+/**
+ * Created by Xiaxing SHI on 11/11/15.
+ */
+public interface StateDelegate {
+    String getPencilXML(Pencil p);
+}
```

Comment:
The pencil (crayon) is asked for every presentation by the subject

getDrawing(PencilState) can easily be used in awt, no ??

(The graphics2D object can be add in the awtDrawing